package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Now my IDE thinks eslint is not installed. Is it possible to make package.json use my global eslint installation? I don't want to install a local copy of eslint just for stylus.